### PR TITLE
Implement mender-configure inventory and Makefile

### PR DESCRIPTION
**Pull request by kacf**
Note that this is submitted without unit tests in order to unblock other people that can potentially work on the feature epic once these core pieces are in place. I will follow up on those afterwards.

### CODE REVIEW

File: src/mender-inventory-mender-configure [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    exit 1
+fi
+
+curl \
+    -Lf \
+    -X PUT \
+    -H "Authorization: Bearer ${AUTH_TOKEN}" \
+    -H "Content-Type: application/json" \
+    -d "${CONFIG}" \
+    "${SERVER}/configuration"
```
Comment (lluiscampos) [inline src/mender-inventory-mender-configure:39]: Maybe good to have the endpoint (which I assume is still WIP) in a variable at the beginning of the script

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+DESTDIR ?= /
```
Comment (lluiscampos) [inline Makefile:1]: Unused. We need it in all install/uninstall commands.

**Review by lluiscampos — APPROVED** [review]
The script looks good! The `sed` commands "work" :)

The Makefile is missing the `DESTDIR`, but pre-approving. Anyway I expect minor adjustments when doing the unit tests.

File: src/mender-inventory-mender-configure (hunk 1 above)
Comment (kacf) [reply src/mender-inventory-mender-configure:39]: Yes, good point, will move it.

File: Makefile (hunk 2 above)
Comment (kacf) [reply Makefile:1]: What do you mean? Should I add that as a comment?

Comment (kacf) [reply Makefile:1]: It is used. Check the next two lines.

Comment (lluiscampos) [reply Makefile:1]: Ouch! Sorry.

I expected it to be used in every single `install` call so I somewhat missed the two following lines :sweat_smile:

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**